src/fine_tune_diffusion.py

```diff
@@ -0,0 +1,5 @@
+from tti_fine_tune.test_text_to_image import TextToImage
+import unittest
+
+if __name__ == "__main__":
+    unittest.main(argv=[''], defaultTest='TextToImage.test_text_to_image', exit = False)
```

Comment:
I know we used this before but can we use just the command line this time, no "unittest" things involved?

The way you wrote it right now we can run the train_text_to_image.py as a command line like

python   tti_fine_tune/train_text_to_image.py
            --pretrained_model_name_or_path BAAI/AltDiffusion-m9
            --train_data_dir dataset_fine_tune
            --mixed_precision fp16
            --allow_tf32
            --gradient_checkpointing
            --resolution 256
            --center_crop
            --random_flip
            --train_batch_size 1
            --gradient_accumulation_steps 1
            --num_train_epochs 2
            --learning_rate 5.0e-09
            --lr_scheduler constant
            --lr_warmup_steps 0
            --output_dir {output_directory}

Reply:
Then we won't need the `class TextToImage(ExamplesTestsAccelerate)` anymore, because that's just a wrapper that runs the command and makes sure that those output files/directories exist